intg/src/main/java/org/apache/atlas/model/discovery/SearchParameters.java

```diff
@@ -48,7 +48,6 @@ public class SearchParameters extends SearchParams implements Serializable {
     private String  termName;
     private String  sortBy;
     private boolean excludeDeletedEntities;
-    private boolean includeClassificationAttributes;
```

Comment:
Don't remove it from here, as this will become a breaking change
We can simply use this param and set it request context and use for indexsearch API or pass it as a parameter to various methods